### PR TITLE
Update app-governance-predefined-policies.md

### PR DESCRIPTION
**Pull request by AndrewVahidnia**

### CODE REVIEW

**Comment by PRMerger10** [comment]
@AndrewVahidnia : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ShannonLeavitt** [comment]
@dcurwin – Can you review the proposed changes? IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

File: CloudAppSecurityDocs/app-governance-predefined-policies.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ App governance contains a set of out of the box policies to detect anomalous app
 
     ![View number of policies.](media/app-governance/number-policies.png)
 
+- You can choose whether you want this policy to apply to all apps in your tenant, specific apps or all apps except the apps you choose. To do so, select Edit scope from the Scope and conditions tab. If you choose specific apps or all apps except for this policy, select Add apps and select the desired apps from the list. In the Choose apps pane, you can select multiple apps to which this policy will be applied. Select Save when you're satisfied with the list.
```
Comment (dcurwin) [inline CloudAppSecurityDocs/app-governance-predefined-policies.md:28]:
```suggestion
- You can choose whether you want this policy to apply to all apps in your tenant, specific apps, or all apps except the apps you choose. To do so, select **Edit scope** from the **Scope and conditions** tab. If you choose specific apps or all apps except for this policy, select **Add apps** and select the desired apps from the list. In the **Choose apps** pane, you can select multiple apps to which this policy will be applied. Select **Save** when you're satisfied with the list.
```

**Comment by PRMerger3** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by dcurwin — APPROVED** [review]

**Comment by AndrewVahidnia** [comment]
#sign-off

**Comment by PRMerger19** [comment]
@AndrewVahidnia: I'm sorry - only the author of this article, @dcurwin, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**